# Generated by Django 3.1.6 on 2021-02-09 21:28

import django.contrib.auth.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('magic_auth', '0005_magicuser_password'),
    ]

    operations = [
        migrations.AddField(
            model_name='magicuser',
            name='username',
            field=models.CharField(default='securepassword123', error_messages={'unique': 'A user with that username already exists.'}, help_text='Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.', max_length=150, unique=True, validators=[django.contrib.auth.validators.UnicodeUsernameValidator()], verbose_name='username'),
            preserve_default=False,
        ),
        migrations.AlterField(
            model_name='magicuser',
            name='password',
            field=models.CharField(default='securepassword123', max_length=64),
        ),
    ]
